homeassistant/components/overkiz/cover_entities/vertical_cover.py

```diff
@@ -41,6 +42,8 @@
     UIClass.WINDOW: CoverDeviceClass.WINDOW,
 }
 
+SERVICE_SET_COVER_POSITION = "set_cover_position_and_tilt"
```

Comment:
This variable seems to be not used.  
You can use it to add it to the supported_features list I think.  
And if it's possible, add it only if the device has the command `OverkizCommand.SET_CLOSURE_AND_ORIENTATION`